packages/blockchain-wallet-v4/src/network/api/misc/index.js

```diff
@@ -34,6 +34,15 @@ export default ({ rootUrl, apiUrl, get, post }) => {
       data: { base: coin, quote: currency, start: start, scale: scale }
     })
 
+  const getPriceTimestampSeries = (coin, currency, txTimestampList) =>
+    post({
+      url: apiUrl,
+      endPoint: `/price/index-series?base=${coin}&quote=${currency}`,
+      contentType: 'application/json',
+      removeDefaultPostData: true,
+      data: txTimestampList
```

Comment:
> removeDefaultPostData

Do you need this if you pass base and quote in data?

Reply:
yes because the POST body expects just an array of timestamps.  Without this flag, the POST body is 
```
{ [], apiKey}
```

https://github.com/blockchain/wiki/wiki/Service-Price-API-Definition#post-priceindex-series